### PR TITLE
email template and created a new verification page

### PR DESCRIPTION
**Pull request by oyindamoladekeye**

### CODE REVIEW

File: backend/templates/ResetPassword/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Heed</title>
+    <link rel="stylesheet" href="index.css">
```
Comment (dprincecoder) [inline backend/templates/ResetPassword/index.html:8]: why am I seeing your changes inside the backend folder?

**Review by dprincecoder — CHANGES_REQUESTED** [review]
please tell me the changes inside backend folder

File: backend/templates/ResetPassword/index.html (hunk 1 above)
Comment (oyindamoladekeye) [reply backend/templates/ResetPassword/index.html:8]: David asked me to put it in backend for email template

Comment (dprincecoder) [reply backend/templates/ResetPassword/index.html:8]: ok

**Pull request merged**